### ISSUE TITLE
pVACvector argument -e 10 doesn't work with MHCflurry

### ISSUE THREAD

**Issue by tryptofan68** [griffithlab/pVACtools#220]
* pvactools version: pvactools==1.1.2
* Python version:  Python 3.5.6
* Operating System: Ubuntu 18.04.1 LTS (on Windows 10)

**Describe the bug**
When using pvacvector with MHCflurry and the `-e` argument of 10, the error: No command specified appears. I have no problem with exactly the same command excpet `-e 9` or using another algorithm.

**To Reproduce**
the following does not work:
```bash
pvacvector run epitopes_list.fa E7NEO H-2-Kb,H-2-Db MHCflurry . -e 10
```

**Output File**
```Using TensorFlow backend.
Executing MHC Class I predictions
Generating Variant Peptide FASTA and Key Files
Generating Variant Peptide FASTA and Key Files - Entries 1-2
Completed
Processing entries for Allele H-2-Db and Epitope Length 10 - Entries 1-2
Running IEDB on Allele H-2-Db and Epitope Length 10 with Method MHCflurry - Entries 1-2
2018-11-06 12:38:31.916093: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
usage: pvacvector [-h]
                  {run,valid_alleles,allele_specific_cutoffs,download_example_data}
                  ...

positional arguments:
  {run,valid_alleles,allele_specific_cutoffs,download_example_data}
    run                 Runs the pVACvector pipeline
    valid_alleles       Shows a list of valid allele names
    allele_specific_cutoffs
                        Show the allele specific cutoffs
    download_example_data
                        Downloads example input and output files

optional arguments:
  -h, --help            show this help message and exit
Error: No command specified
```

**Expected behavior**
The following works:
```bash
pvacvector run epitopes_list.fa E7NEO H-2-Kb MHCflurry . -e 8
```
```
Using TensorFlow backend.
Executing MHC Class I predictions
Generating Variant Peptide FASTA and Key Files
Generating Variant Peptide FASTA and Key Files - Entries 1-2
Completed
Processing entries for Allele H-2-Kb and Epitope Length 8 - Entries 1-2
Running IEDB on Allele H-2-Kb and Epitope Length 8 with Method MHCflurry - Entries 1-2
2018-11-06 12:29:17.512553: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Completed
Parsing IEDB Output for Allele H-2-Kb and Epitope Length 8 - Entries 1-2
Completed
Graph contains 3 nodes and 6 edges.
Finding path.
 Temperature        Energy    Accept   Improve     Elapsed   Remaining
250000000.00000      26818.58   100.00%    33.40%     0:00:00     0:00:00
26818 distance :
         E7epitope
         OVAI
         OVAII
```
No problem neither with 
```bash
pvacvector run epitopes_list.fa E7_NEO H-2-Kb,H-2-Db NetMHCcons . -e 8,9,10
```

**Comment by tryptofan68** [comment]
OK. Sure please give a try with this file
[epitopes_list.zip](https://github.com/griffithlab/pVACtools/files/2553601/epitopes_list.zip)

Thank you for your support!

**Comment by susannasiebert** [comment]
I made a pull request for your issue. Until this fix goes live you can get around this issue by picking a `-e` parameter that is <= to your shortest fasta sequence. In your example, your shortest fasta sequence is 8 amino acids long. I was able to run your file through pVACvector with MHCflurry using `-e 8`.

**Comment by tryptofan68** [comment]
OK, yes pVACvector works well with `-e 8,9` + MHCflurry.
Why does it work with the same sequences and `-e 10` + e.g. NetMHC but not MHCflurry?
Thanks for your support.

**Comment by susannasiebert** [comment]
Thank you for asking. It caused me to investigate further and discover the following:
- Underneath the hood we create peptides that combined your input epitopes with spacer sequences as well as each other. Sometimes those are shorter than the requested `-e` epitope length.
- This would cause MHCflurry to fail because we wouldn't be able to calculate epitopes for these sequences.
- NetMHC gets called using the IEDB API. However, IEDV would silently skip these short sequences. This is the reason it wouldn't cause an error. However now that I investigated this issue more I discovered that when it skips sequences, it also relabels the remaining epitopes resulting in a mismatch between the IEDB provided labels and ours which results in incorrect downstream processing. This is another bug that I will need to fix. Stay tuned.

**Comment by susannasiebert** [comment]
This issues has been fixed in version 1.1.3. @tryptofan68 please update to the latest version and see if that fixes the error you were seeing. I will resolve this issue but feel free to reopen or make a new issue should you still run into error.
